library/common/extensions/filters/http/network_configuration/filter.proto

```diff
@@ -3,4 +3,7 @@ syntax = "proto3";
 package envoymobile.extensions.filters.http.network_configuration;
 
 message NetworkConfiguration {
+  // If set to true, the filter will permit the NetworkConfigurator to provide upstream
+  // socket option that MAY bind a connection to a specific network interface.
```

Comment:
should we expand on what `MAY` means here? As in - what needs to happen for the binding to happen? Or maybe that would be to specific for a doc string?

Reply:
I actually left it ambiguous intentionally since whether it does will change based on configuration and the heuristic we choose.